{"uuid": "d29b9db2-d015-49ab-a987-0e8b7c1cd8c2", "children": ["f42480c1-2469-4047-94a7-d8999466f48b"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: Failed to read marionette port\n\n", "trace": "  File \"C:\\Users\\Galina_Kilovataia\\yandex\\diploma_3\\.venv\\lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Users\\Galina_Kilovataia\\yandex\\diploma_3\\.venv\\lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"C:\\Users\\Galina_Kilovataia\\yandex\\diploma_3\\.venv\\lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\Galina_Kilovataia\\yandex\\diploma_3\\.venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 1125, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\Galina_Kilovataia\\yandex\\diploma_3\\.venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 880, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\Users\\Galina_Kilovataia\\yandex\\diploma_3\\conftest.py\", line 19, in driver\n    browser = webdriver.Firefox(options=firefox_options)\n  File \"C:\\Users\\Galina_Kilovataia\\yandex\\diploma_3\\.venv\\lib\\site-packages\\selenium\\webdriver\\firefox\\webdriver.py\", line 71, in __init__\n    super().__init__(command_executor=executor, options=options)\n  File \"C:\\Users\\Galina_Kilovataia\\yandex\\diploma_3\\.venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 208, in __init__\n    self.start_session(capabilities)\n  File \"C:\\Users\\Galina_Kilovataia\\yandex\\diploma_3\\.venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 292, in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n  File \"C:\\Users\\Galina_Kilovataia\\yandex\\diploma_3\\.venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 347, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\Galina_Kilovataia\\yandex\\diploma_3\\.venv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1719843551738, "stop": 1719843758070}], "start": 1719843551738, "stop": 1719843765796}